### PR TITLE
Fixes #19584: when the password are updated by the policies, rudder-jetty and rudder-slapd are not restarted

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/19584

### CODE REVIEW

File: techniques/system/rudder-service-webapp/1.0/password-check.cf [hunk 1]
```diff
@@ -50,4 +55,5 @@ bundle agent system_rudder_webapp_properties {
       "any" usebundle => _method_reporting_context("Check webapp configuration", "webdav password");
       "any" usebundle => file_key_value_present("${properties_file}", "rudder.webdav.password", "${dav_password[2]}", "=");
 
+file_key_value_present__opt_rudder_etc_rudder_web_properties_repaired
```
Comment (amousset) [inline techniques/system/rudder-service-webapp/1.0/password-check.cf:58]: looks like a copy/paste fail

Comment (ncharles) [reply techniques/system/rudder-service-webapp/1.0/password-check.cf:58]: TOTALLY

**Comment by ncharles** [comment]
PR updated with a new commit

**Comment by ncharles** [comment]
don't merge, it doesn't restart rudder-relayd

**Comment by ncharles** [comment]
PR updated with a new commit

File: techniques/system/server-common/1.0/reloadRudderServices.st [hunk 2]
```diff
@@ -46,7 +46,7 @@ bundle agent system_reload_rudder_services {
     rudder_server_system_restart_apache::
       "any" usebundle => service_reload("${apache_service_name}");
     rudder_server_system_restart_relayd::
-      "any" usebundle => service_reload("${relayd_service_name}");
+      "any" usebundle => service_start("${relayd_service_name}");
```
Comment (amousset) [inline techniques/system/server-common/1.0/reloadRudderServices.st:49]:
```suggestion
      "any" usebundle => service_restart("${relayd_service_name}");
```

**Comment by ncharles** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Pull request merged**